### PR TITLE
add tests and fixup behavior for connections with no first/last

### PR DESCRIPTION
**Pull request by tberman**
This adds some basic tests, and fixes the issue when no first/last is passed in.

### CODE REVIEW

File: spec/graphql/relay/relation_connection_spec.rb [hunk 1]
```diff
@@ -84,6 +87,12 @@ def get_last_cursor(result)
       assert_equal(["Death Star"], get_names(result))
     end
 
+    it 'works without first/last/after/before' do
+      result = query(query_string)
+
+      assert_equal(false, result["data"]["empire"]["bases"]["pageInfo"]["hasNextPage"])
```
Comment (rmosolgo) [inline spec/graphql/relay/relation_connection_spec.rb]: I think we should also test a few more things in this case: 

- If there is a max-page-size, it is applied
- If there's no max-page-size, all items are returned

**Comment by rmosolgo** [comment]
Thanks for taking a look at this! Seems like I misread the spec at first. This will also address #1

**Comment by tberman** [comment]
@rmosolgo added some more tests, great call, found some interesting bugs!

**Comment by rmosolgo** [comment]
nice, thanks for the fix!

**Comment by rmosolgo** [comment]
shipped in 0.9.1!

**Pull request merged**